### PR TITLE
BoundingSphere docs + few other doc fixes

### PR DESCRIPTION
**Pull request by Shqrdx**
Docs for BoundingSphere. I didnt change the code inside methods so no tests needed. +Added new overload for Contains..

### CODE REVIEW

**Comment by Shqrdx** [comment]
This PR is ok ?

**Comment by Shqrdx** [comment]
@tomspilman This PR is ok ?

File: MonoGame.Framework/BoundingSphere.cs [hunk 1]
```diff
@@ -4,55 +4,73 @@
...
+        #endregion
+
+        #region Internal Properties
+
+        internal string DebugDisplayString
+        {
+            get
+            {
+                return string.Concat(
+                    "Pos( ", this.Center.DebugDisplayString, " )  \r\n",
```
Comment (Shqrdx) [inline MonoGame.Framework/BoundingSphere.cs]: I think Pos in DebugDisplayString is incorrect and should be changed to Center... what do you think @tomspilman, @KonajuGames ?

Comment (KonajuGames) [inline MonoGame.Framework/BoundingSphere.cs]: The property is called Center, so it makes sense.

**Comment by KonajuGames** [comment]
> I didnt change the code inside methods so no tests needed.

Even just moving methods within the class, such as Transform in this case, makes the diff much more difficult to read and see actual changes.

I think this PR is ok, and if you make the change to DebugDisplayString I think it can be merged.

**Comment by KonajuGames** [comment]
Thanks @Shqrdx . Merging.

File: MonoGame.Framework/BoundingSphere.cs [hunk 2]
```diff
@@ -4,55 +4,73 @@
...
-using System.Globalization;
-using System.Runtime.Serialization;
 using System.Diagnostics;
+using System.Runtime.Serialization;
 
 namespace Microsoft.Xna.Framework
 {
-
+    /// <summary>
+    /// Describes a virtual sphere in 3D-space for bounding operations.
```
Comment (tomspilman) [inline MonoGame.Framework/BoundingSphere.cs]: Remove the word 'virtual'.

File: MonoGame.Framework/BoundingSphere.cs [hunk 3]
```diff
@@ -4,55 +4,73 @@
...
-
-        public void Transform(ref Matrix matrix, out BoundingSphere result)
-        {
-            result.Center = Vector3.Transform(this.Center, matrix);
-            result.Radius = this.Radius * ((float)Math.Sqrt((double)Math.Max(((matrix.M11 * matrix.M11) + (matrix.M12 * matrix.M12)) + (matrix.M13 * matrix.M13), Math.Max(((matrix.M21 * matrix.M21) + (matrix.M22 * matrix.M22)) + (matrix.M23 * matrix.M23), ((matrix.M31 * matrix.M31) + (matrix.M32 * matrix.M32)) + (matrix.M33 * matrix.M33)))));
-        }
+        #region Contains
 
+        /// <summary>
+        /// Gets how the specified <see cref="BoundingBox"/> lie within the bounds of this sphere.
```
Comment (tomspilman) [inline MonoGame.Framework/BoundingSphere.cs]: `Test if a bounding box is fully inside, outside, or just intersecting the sphere.`

File: MonoGame.Framework/BoundingSphere.cs [hunk 4]
```diff
@@ -175,6 +233,15 @@ public void Contains(ref Vector3 point, out ContainmentType result)
                 result = ContainmentType.Intersects;
         }
 
+        #endregion
+
+        #region CreateFromBoundingBox
+
+        /// <summary>
+        /// Creates the smallest <see cref="BoundingSphere"/> that can surround a specified <see cref="BoundingBox"/>.
+        /// </summary>
```
Comment (tomspilman) [inline MonoGame.Framework/BoundingSphere.cs]: Use `contain` and not `surround`.

File: MonoGame.Framework/BoundingSphere.cs [hunk 5]
```diff
@@ -182,6 +249,11 @@ public static BoundingSphere CreateFromBoundingBox(BoundingBox box)
             return result;
         }
 
+        /// <summary>
+        /// Creates the smallest <see cref="BoundingSphere"/> that can surround a specified <see cref="BoundingBox"/>.
```
Comment (tomspilman) [inline MonoGame.Framework/BoundingSphere.cs]: Use `contain` and not `surround`.

File: MonoGame.Framework/BoundingSphere.cs [hunk 6]
```diff
@@ -195,11 +267,23 @@ public static void CreateFromBoundingBox(ref BoundingBox box, out BoundingSphere
             result = new BoundingSphere(center, radius);
         }
 
+        #endregion
+
+        /// <summary>
+        /// Creates the smallest <see cref="BoundingSphere"/> that can surround a specified <see cref="BoundingFrustum"/>.
```
Comment (tomspilman) [inline MonoGame.Framework/BoundingSphere.cs]: Use `contain` and not `surround`.

**Comment by Shqrdx** [comment]
@tomspilman I've added https://github.com/mono/MonoGame/pull/3992 for fixing these issues..

**Pull request merged**